openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -139,10 +139,10 @@ <h3 class="message__title">{% trans "We could not submit your response" %}</h3>
 
                 <ul class="list list--actions">
                     <li class="list--actions__item">
-                        <a aria-role="button" href="#" id="step--response__submit"
-                            class="action action--submit step--response__submit {{ submit_enabled|yesno:",is--disabled" }}">
+                        <button type="submit" id="step--response__submit"
```

Comment:
@cptvitamin I converted this anchor tag to a button. I added type="submit" because other similar buttons had that. Let me know if it is incorrect.